MITADS/opensubtitles_exporter.py

```diff
@@ -121,22 +122,19 @@ def parsexmlfile(xml_path, count_file):
 count_file = 0
 
 # Parse 5 files at once
-pool = ThreadPoolExecutor(max_workers=20)
+pool = ProcessPoolExecutor(max_workers=20)
 
-total_lines = 0
-for xml_path in pathlist:
+
+paths = []
+for i, xml_path in enumerate(pathlist):
   year_folder = str(xml_path.parent.parent._parts[len(xml_path.parent.parent._parts)-1])
   year_folder_int = int(year_folder)
   if(year_folder_int<start_year):
       continue
+  paths.append((str(xml_path), i))
 
-  xml_path = str(xml_path)
-  future = pool.submit(parsexmlfile, xml_path, count_file)
-  total_lines += future.result()
-  #parsexmlfile(xml_path, count_file)
-  print(' Actual lines ' + str(total_lines))
-
-  count_file +=1
 
+total_lines = pool.map(parsexmlfile, paths)
+total_lines = sum(list(total_lines))
```

Comment:
Nit: `sum(total_lines)` should be equivalent and avoid creating an intermediate list